config/filter.d/apache-pass.conf

```diff
@@ -0,0 +1,20 @@
+# Fail2Ban Apache pass filter
+# This filter is for access.log, NOT for error.log
+#
+# The knocking request must have a referer.
+
+[INCLUDES]
+
+before = apache-common.conf
+
+[Definition]
+
+failregex = ^<HOST> - \w+ \[\] "GET <knocking_url> HTTP/1\.[01]" 200 \d+ ".*" "[^-].*"$
+
+ignoreregex =
+
+[Init]
+
+knocking_url = /knocking/
```

Comment:
That's great! I'm trying to help you....you can test locally by running setup.py test or bin/fail2ban-testcases....If you move the declaration of `knocking_url` to `[Definition]`, I'm wondering if it will be better?